modules/webxr/webxr_interface_js.cpp

```diff
@@ -420,7 +419,7 @@ void WebXRInterfaceJS::_update_tracker(int p_controller_id) {
 		float *tracker_matrix = godot_webxr_get_controller_transform(p_controller_id);
 		if (tracker_matrix) {
 			Transform transform = _js_matrix_to_transform(tracker_matrix);
-			tracker->set_position(transform.origin);
+			tracker->set_position(transform.origin * arvr_server->get_world_scale());
```

Comment:
In Godot 3 there as actually a `set_rw_position` that sets the position before scaling (as that is what is held within), `set_position` expects a scaled position but will reverse the scaling internally.

So it's nicer to call `set_rw_position` here.

In Godot 4 I'm only dealing with "real world" position (rw ;) ) within the trackers to prevent this confusion, and scaling is done by the XRServer/XRNodes.

Reply:
Thanks! I've switched to using `set_rw_position()`